File: sunbin/day20/002_문자열정렬하기1.js
// https://school.programmers.co.kr/learn/courses/30/lessons/120850

function solution(my_string) {
  const answer = my_string
    .split("")
    .map(Number)
    .filter((item) => !isNaN(item))
    .sort((a, b) => a - b);
  return answer;
}

File: sunbin/Lv0/day20/python/002_문자열정렬하기1.py
# https://school.programmers.co.kr/learn/courses/30/lessons/120850

import re


def solution(my_string):
    numbers = list(re.sub('[^0-9]', '', my_string))
    num_type_list = sorted([int(i) for i in numbers])

    return num_type_list


# 다른 풀이: 한 줄 풀이
def solution_b(my_string):
    answer = sorted([int(c) for c in my_string if c.isdigit()])

    return answer


solution("hi12392")
